api-inference-community/docker_images/fairseq/app/pipelines/text_to_speech.py

```diff
@@ -11,7 +12,9 @@ class TextToSpeechPipeline(Pipeline):
     def __init__(self, model_id: str):
         # hardcoded stuff can later be moved to a config
         model, cfg, task = load_model_ensemble_and_task_from_hf_hub(
-            model_id, arg_overrides={"vocoder": "griffin_lim", "fp16": False}
+            model_id,
+            arg_overrides={"vocoder": "griffin_lim", "fp16": False},
+            cache_dir=os.getenv("HUGGINGFACE_HUB_CACHE"),
```

Comment:
Use `HUGGINGFACE_HUB` to make docker it the cache (See the dockerfile this is defined as `/data` and this is mounted as an external disk on the kube cluster to make the model files persistent.

Reply:
Thanks for the explanation